index.js

```diff
@@ -80,7 +80,12 @@ function buildCache (client, opts) {
           return asPromise(client.zrem.bind(client), ZSET_KEY, key)
             .then(() => null);
         }
-        return JSON.parse(results[0]);
+
+        if (Array.isArray(results[0])) {
```

Comment:
looks like you are doing this check twice, can you extract it to a function?

Reply:
> Can you add a commented out line like [this one](https://github.com/facundoolano/redis-lru/blob/master/test/index.js#L5) for ioredis, and make sure tests pass if you use it with a running redis instance?

I tried doing this with the code as is and several tests fail. I'm finding no discernable difference but tests seem to work if I make the max size of the LRU one bigger then the tests originally had. For some reason, the first item (I'm looking at the get or set tests max size test) `k1` is always null unless I bump the size of the LRU up from 3 to 4. Any ideas?